### ISSUE TITLE
Evaluate on the test set

### ISSUE THREAD

**Issue by arekpaterak** [michalszc/ml-for-science-articles#13]
After all experiments.

For all our methods:
- CountVectorizer
- TfidfVectorizer
- Doc2Vec
- pretrained models
  - all-MiniLM-L6-v2
  - SPECTER2
  - all-mpnet-base-v2 (optionally)
- finetuned model
  - all-MiniLM-L6-v2
- reranker on the top of the all-MiniLM-L6-v2 (work in progress)
